install/createDefaultPassword.sh

```diff
@@ -18,24 +34,40 @@ echo "Please enter a default password that can be used to login into the default
 echo "This password will be stored in the created project war as a hash. "
 echo "This is technically secure, but it is recommended to change the password on installation"
 
-htpasswd -cBC 12 ${resourcesDir}/adminPassword.txt admin 
+echo "Output location: ${resourcesDir}"
```

Comment:
@CalebSLane @mozzy11 the logic in this file can probably be simplified, but I wanted to add a way to easily run this as a non-interactive command while keeping the same exact behavior as before

Reply:
@CalebSLane If we wanted to use the built docker image in a native installation of OpenELIS, should we run this script externally and then mount the password file into the docker image? How would we be able to use this approach in general with pre-built docker images?

Reply:
@mozzy11 in a docker install, we might still want to provide user-input options. We would want the user to be able to supply the admin password in some way. Currently, I don't think we are able to use a pre-built image and supply a user password, since the password is set in the build stage. @CalebSLane is that correct?

Reply:
@pmanko , 
may be for the case when a user wants to build the docker image form source code , we could run the script outside the docker build

Reply:
No i dont think its possible to set the default password after the image is build.
but i dont see any big issue with setting a specific default password as this can simply be changed on Login.
OpenMRS has a fixed default password .

Reply:
May be the other way i think we can set a default password after the docker build , 
we can just mount an sql script file into the db-container to directly insert the user and passowrd into the db container

Reply:
good points - maybe we can get @CalebSLane 's thoughts on it. Let's not block this PR in the meantime, since we can just reset the password on login.

Reply:
sure, makes sense